Trello_items.py

```diff
@@ -0,0 +1,78 @@
+from secrets import KEY, TOKEN
+import requests
+
+todo_boardid = "5f6076c34c5f48265943e31e"
+todolistid = "5f6076e68cc021208da06d2b"
+donelistid = "5f6076e96994a166c05385a6"
+
+apiurl = "https://api.trello.com/1/"
+boardsurl = apiurl + 'boards/'
+getcardsonboardsurl = boardsurl + todo_boardid + '/cards'
+cardsurl = apiurl + 'cards/'
+
+def getcardsonlist_URL(cardid):
+    URL = cardsurl + cardid + "/list"
+    return URL
+def putcardsonlist_URL(cardid):
+    URL = cardsurl + cardid
+    return URL
+
+def buildquery(querytype, title=''):
+    if querytype == 'getitems':
+        return {'key' : KEY, 'token' : TOKEN}
+    elif querytype == "saveitem":
+        return {'key' : KEY, 'token' : TOKEN, "idList": donelistid}
+    elif querytype == "additem":
+        return {'key' : KEY, 'token' : TOKEN, "idList": "5f6076e68cc021208da06d2b", "name" : title}
+
+class Item:
+    def __init__(self, id, title, status='To Do'):
+        self.id = id
+        self.status = status
+        self.title = title
+
+
+def get_items_trello():
+    """
+    Fetches all cards from the Trello.
+
+    Returns:
+        list: The list of saved items.
+    """
+    cards = requests.get(getcardsonboardsurl, params=buildquery('getitems'))
+    cards_json = cards.json()
+
+    items = []
+    for card in cards_json:
+        cardlist = requests.get(getcardsonlist_URL(card['id']), params=buildquery('getitems'))
+        cardlist_json = cardlist.json()
+        if cardlist_json['name'] == 'Done':
+            cardstatus = 'Completed'
+        else :
+            cardstatus = cardlist_json['name'] 
+        items.append(Item(card['id'], card['name'], cardstatus))
+    return items
+
+
+def save_item_trello(id):
```

Comment:
The function name is misleading. It's actually very specifically updating a card to status "Done" but sounds like a generic function for saving an item.